kubetest2/pkg/app/testers/standard/ginkgo/ginkgo.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ginkgo
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"k8s.io/test-infra/kubetest2/pkg/app/testers"
+	"k8s.io/test-infra/kubetest2/pkg/app/testers/standard/ginkgo/kubectl"
+	"k8s.io/test-infra/kubetest2/pkg/exec"
+	"k8s.io/test-infra/kubetest2/pkg/types"
+
+	"github.com/spf13/pflag"
+)
+
+const (
+	binary = "ginkgo" // TODO(RonWeber): Actually find these binaries.
+
+)
+
+var (
+	kubeRoot     string
+	platformPath string
+)
+
+const usage = `--flake-attempts Make up to this many attempts to run each spec.
+--parallel Run this many tests in parallel at once.  Defaults to 1 (Not parallel)
+--skip Regular expression of jobs to skip.`
+
+func init() {
+	testers.Register("ginkgo", usage, NewTester)
+}
+
+// Tester implements a kubetest2 types.Tester that exec's it's arguments
+type Tester struct {
+	conformanceTest bool
+
+	flakeAttempts string
+	parallel      string
+	skipRegex     string
+
+	host           string
+	kubeconfigPath string
+	provider       string
+
+	deployer types.Deployer
+}
+
+// NewTester creates a new Tester
+func NewTester(common types.Options, testArgs []string, deployer types.Deployer) (types.Tester, error) {
+	wd, _ := os.Getwd()
+	kubeRoot = filepath.Join(wd, "kubernetes")                            //TODO(RonWeber): Real logic to get kubernetes
+	platformPath = filepath.Join(kubeRoot, "platforms", "linux", "amd64") //TODO(RonWeber): Real logic to figure out current platform.
+
+	t := Tester{}
+	t.deployer = deployer
+
+	flags := bindFlags(&t)
+	flags.Parse(testArgs)
+
+	t.conformanceTest = true //TODO(RonWeber): Better logic here.
+
+	return t, nil
+}
+
+func bindFlags(t *Tester) *pflag.FlagSet {
+	flags := pflag.NewFlagSet("ginkgo tester", pflag.ContinueOnError)
+	flags.StringVar(&t.flakeAttempts, "flake-attempts", "1", "Make up to this many attempts to run each spec.")
+	flags.StringVar(&t.parallel, "parallel", "1", "Run this many tests in parallel at once.  Defaults to 1 (Not parallel)")
```

Comment:
hummm intVar?

Reply:
We're just pushing the string literally into ginkgo args. My thought process was to err on the side of not validating the args here, because maybe a special value will need to be passed through at some point.